Sources/Pathspec/GitIgnoreSpec.swift

```diff
@@ -8,14 +8,25 @@
 import Foundation
 
 struct GitIgnoreSpec: Spec {
+    enum Error: Swift.Error {
+        case emptyPattern
+        case containsPoundPrefix
```

Comment:
```suggestion
        case commented
```
something along these lines maybe? (`commented` or maybe `commentedOut` or `disabled`)